tests/core/cmds/test_keys.py

```diff
@@ -302,7 +366,14 @@ def test_delete(self, tmp_path, empty_keyring, mnemonic_seed_file):
 
         runner = CliRunner()
         init_result: Result = runner.invoke(
-            cli, ["--root-path", os.fspath(tmp_path), "--keys-root-path", os.fspath(keys_root_path), "init"]
+            cli,
+            [
```

Comment:
another one here. I won't mark any more because that would be kinda dumb but please fix all of them.